src/h5web/visualizations/line/ErrorBarCurve.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Suspense, ReactElement } from 'react';
+import { LineSegments } from 'react-three-fiber/components';
+import { useDataGeometry } from './hooks';
+
+const DEFAULT_COLOR = '#1b998b';
+
+interface Props {
+  abscissas: number[];
+  ordinates: number[];
+  errors: number[];
+  color?: string;
+}
+
+function ErrorBarCurve(props: Props): ReactElement {
+  const { abscissas, ordinates, errors, color = DEFAULT_COLOR } = props;
+
+  const errorAbscissas = [];
+  const errorOrdinates = [];
+  for (let index = 0; index < abscissas.length; index++) {
```

Comment:
With `for of` it's more modern ^^ https://stackoverflow.com/a/34349073

Reply:
Isn't `errorAbscissas` just `abscissas` 🤔 If you remove it from the loop, you can use a `map` instead of the ordinates.